measurements/honest/README.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Honest
+emoji: 🤗
+colorFrom: blue
+colorTo: green
+sdk: gradio
+sdk_version: 3.0.2
+app_file: app.py
+pinned: false
+tags:
+- evaluate
+- measurement
+description: >-
+The HONEST score aims to measure hurtful sentence completions in language models.
+---
+
+# Measurement Card for HONEST
+
+## Measurement description
+The HONEST score aims to measure hurtful sentence completions in language models.
+We score uses HurtLex, a multilingual lexicon of hurtful language, to evaluate the completions.
+It goals is to quantify how many times a sentence is completed with a hurtful word and if there is a difference between
+groups (e.g. genders, sexual orientations, etc.).
+
+## How to use
+
+When loading the model, specify the language of the prompts and completions.
+The available languages are: 'it' (Italian), 'fr' (French), 'es' (Spanish), 'pt' (Portuguese), 'ro' (Romanian), 'en' (English).
+```python
+>>> honest = evaluate.load('measurements/honest', 'en')
+```
+Arguments:
+    **predictions** (list of list of `str`): a list of completions to [HONEST prompts](https://huggingface.co/datasets/MilaNLProc/honest)
```

Comment:
Since we never enforce this, the metric is a little bit different in the case that they pass in random completions; it won't truly be an HONEST score. Maybe we can add a comment to the description of the metric above to make this clear? Not much we can do otherwise besides warning the user a lot about proper use :)

Reply:
I did that in the description, do you think it's clear now?

Reply:
Yes, that looks great!